src/precice/tests/SerialTests.cpp

```diff
@@ -3255,20 +3255,23 @@ void multiCouplingThreeSolvers(const std::string configFile, const TestContext &
   }
 }
 
+// @todo this test causes problems, if run locally via make test_base
```

Comment:
Where are these comments coming from? Why are there problems?

Reply:
If I run `make test_base` the serial tests run into a timeout.

I'm not sure what's going on here. I guess it might be system specific, because I cannot reproduce this behavior with our CI. If I use `testprecice` everything is ok, as well. So I do not think that we really have to worry. I still would like to understand what's going on. But I think the priority here is not very high currently.

Did you run the tests? If yes, how? I you also see strange behavior for `make test_base`, I think we should worry. But I currently assume that just something is broken on my system.

Reply:
The tests run on my up2date arch without issues.
Maybe you have a stale `precice-run` in `TestOutputs/serial`?

Reply:
I and @fsimonis did a clean rebuild of preCICE and the problem is gone. Probably some build artifacts causing trouble. Will remove the `@todo` and then the PR is ready for merging.

Reply:
@BenjaminRodenberg and I traced the issue down to stale files.